src/api.js

```diff
@@ -1,10 +1,12 @@
 import axios from "axios";
 
-export const getArticles = () => {
-  const response = axios
-    .get("https://articles-service-api.onrender.com/api/articles")
-    .then((articles) => articles.data);
+export const getArticles = (articleId) => {
+  let urlStr = "https://articles-service-api.onrender.com/api/articles";
+
+  if (articleId !== undefined) {
+    urlStr += `/${articleId}`;
```

Comment:
Rather than having this logic in your `getArticles` function, you should have a separate API function (called something like `getSingleArticle`) in here.